app/code/Magento/NewRelicReporting/Model/NewRelicWrapper.php

```diff
@@ -67,6 +67,19 @@ public function setTransactionName(string $transactionName): void
         }
     }
 
+    /**
+     * Wrapper for 'newrelic_end_transaction'
+     *
+     * @param bool $ignore
+     * @return void
+     */
+    public function endTransaction($ignore = false)
```

Comment:
Unfortunately, we are not allowed to add new public methods according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
Does it mean that I can throw this PR to the trash as long as we don't have 2.4-dev branch?